src/components/Item/Item.js

```diff
@@ -1,49 +1,51 @@
 import React from "react";
-import Card from "@material-ui/core/Card";
-import CardActionArea from "@material-ui/core/CardActionArea";
-import CardActions from "@material-ui/core/CardActions";
-import CardContent from "@material-ui/core/CardContent";
-import CardMedia from "@material-ui/core/CardMedia";
-import Button from "@material-ui/core/Button";
 import img from '../images/nike.jpg'
 import './Item.scss'
+import LocalMallOutlinedIcon from '@material-ui/icons/LocalMallOutlined';
+import {Link} from "react-router-dom";
```

Comment:
`import { Link } from "react-router-dom"` <- whitespaces needed for better readability